build-scripts/relabel_ids.py

```diff
@@ -23,16 +23,18 @@ def parse_args():
     parser = argparse.ArgumentParser(
         description="This script finds check-content files (currently, "
         "OVAL and OCIL) referenced from XCCDF and synchronizes all IDs.")
-    parser.add_argument("xccdf_file")
-    parser.add_argument("new_xccdf_file")
-    parser.add_argument("id_name", help="ID naming scheme")
+    parser.add_argument("--input-xccdf", help="Input XCCDF file")
```

Comment:
This change, according to the `argparse` design, indicate the `input-xccdf` is optional when `--` or `-` is used. I suggest to use the [required](https://docs.python.org/3/library/argparse.html#required) keyword or use a [default](https://docs.python.org/3/library/argparse.html#default) keyword to set a default value to avoid runtime errors.